projector-html-runtime-purs/bower.json

```diff
@@ -24,7 +23,8 @@
     "purescript-strings": "^1.0.0",
     "purescript-tailrec": "^1.0.0",
     "purescript-tuples": "^1.0.0",
-    "purescript-unfoldable": "^1.0.0"
+    "purescript-unfoldable": "^1.0.0",
+    "purescript-pux": "^6.0.1"
```

Comment:
As discussed offline, be good to keep pux out of this if possible, it's not likely to be our only choice for long.